omero/export_scripts/Batch_ROI_Export.py

```diff
@@ -278,18 +299,19 @@ def batch_roi_export(conn, script_params):
 
     # Write to csv
     file_ann = write_csv(conn, export_data, script_params, symbol)
-    if script_params['Data_Type'] == "Dataset":
-        datasets = conn.getObjects("Dataset", script_params['IDs'])
-        link_annotation(datasets, file_ann)
-    else:
+    if dtype == "Image":
         link_annotation(images, file_ann)
+    else:
+        objects = conn.getObjects(dtype, script_params['IDs'])
+        link_annotation(objects, file_ann)
     message = "Exported %s shapes" % len(export_data)
     return file_ann, message
 
 
 def run_script():
     """The main entry point of the script, as called by the client."""
-    data_types = [rstring('Dataset'), rstring('Image')]
+    data_types = [rstring(s) for s in
```

Comment:
This is much more than Screen as indicated in the PR title.
All containers type need to be reviewed

Reply:
Yes, that is in the description. Not space in the title for all testing instructions.